src/openrct2/platform/windows.c

```diff
@@ -1005,4 +1007,15 @@ bool platform_setup_uri_protocol()
 
 ///////////////////////////////////////////////////////////////////////////////
 
+float platform_get_default_scale()
+{
+    SetProcessDpiAwareness(PROCESS_SYSTEM_DPI_AWARE);
+    uint32 dpi = GetDpiForSystem();
+    if (dpi > 96)
+    {
+        return 2.0;
```

Comment:
Why can't you just return `dpi / 96.f`? We have "Smooth NN" (#6588) implemented which provides nice user experience even on non-integer scales and we ship it this way on Android.

I also think returning just two values is rather limiting, devices like https://www.notebookcheck.net/HP-Spectre-x360-13-7500U-4K-UHD-Convertible-Review.210812.0.html have 13" 2160p screens, that's around 330ppi, so easily that should at least be 3 to be usable in such case.